product_docs/docs/pgd/6/concepts/nodes-and-groups.mdx

```diff
@@ -25,11 +25,10 @@ Each group selects it's own RAFT leader from the group members.
 If the group is a data group that is made up of data nodes it also uses RAFT to elect a write leader node for that group.
 The write leader node will be sent all the read/write client connections for that group and will be the node that handles all write operations for that group, assuming that the client connections come in through the connection manager of a node in that group.
 
-!!! RAFT
-    RAFT is a consensus algorithm that is used to ensure that all nodes in a group agree on the state of the group.
-    It allows a group of nodes to elect a leader node, and to ensure that all nodes in the group are in sync with each other over decisions.
-    The most important thing to know about RAFT is that it needs an odd number of nodes in any group to function correctly.
-    That's because RAFT uses a majority vote between the nodes to agree on the state of the group.
+RAFT is a consensus algorithm that is used to ensure that all nodes in a group agree on the state of the group.
```

Comment:
I think this was supposed to be some type of markdown section.

```
[08/20/25 11:00:49] mbp ➜  docs git:(dev/pe/dump-restore) rg '!!!' | wc
    7534   15995  637613
```

Reply:
Good point. I changed to !!!info as that seems most appropriate.